### PR TITLE
Billing Job - Add Stored Procedures

### PR DESCRIPTION
**Pull request by nreal**
# Background
Doppler Billing Job should be executed one day per month then the Job executes the next six stored procedures with Dapper for send user billing dato to SAP system:
[dbo].[SAP_CM_GB_BISIDE_ARG]
[dbo].[SAP_CM_GB_BISIDE_USD]
[dbo].[SAP_CM_GB_DD_USD]
[dbo].[SAP_GB_BISIDE_ARG]
[dbo].[SAP_GB_BISIDE_USD]
[dbo].[SAP_GB_DD_ARG]

Can you Review It?

# Pending Task
- [ ] Unit Test

### CODE REVIEW

File: CrossCutting/DopplerSapService/Entities/UserBilling.cs [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+        public string Periodicity { get; set; }
+        public string PeriodMonth { get; set; }
+        public string PeriodYear { get; set; }
+        public string PlanFee { get; set; }
+        public string Discount { get; set; }
+        public string ExtraEmails { get; set; }
+        public decimal? ExtraEmailsFeePerUnit { get; set; }
+        public string ExtraEmailsPeriodMonth { get; set; }
+        public string ExtraEmailsPeriodYear { get; set; }
+        public string ExtraEmailsFee { get; set; }
```
Comment (fgchaio) [inline CrossCutting/DopplerSapService/Entities/UserBilling.cs:20]: Why string? shouldn't it be int?

Comment (nreal) [reply CrossCutting/DopplerSapService/Entities/UserBilling.cs:20]: Yes is possible change `string ` to `int?`,  I'll do

Comment (nreal) [reply CrossCutting/DopplerSapService/Entities/UserBilling.cs]: Done

**Comment by nreal** [comment]
@MarianoHerrera Ok, the Stored procedures will be made by @fgchaio in other PR, thanks

**Pull request merged**